assembl/static2/css/components/posts.scss

```diff
@@ -581,20 +617,28 @@ $deleted-post-border: 1px solid $light-grey !default;
       .sentiments {
         border-top: 0;
         display: flex;
-        flex-direction: column;
+        flex-direction: row;
```

Comment:
Horizontal display of the sentiments now ? Following the suggestion of Louisa & Florence ?